### PR TITLE
Gallery preview

### PR DESCRIPTION
**Pull request by pateva**

### CODE REVIEW

File: ics-frontend/src/app/single-image-page/single-image-page.component.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <div class="container">
 <mat-card class="card">
-    <img mat-card-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" alt="Photo of a Shiba Inu">
+    <img mat-card-image [src]="imageUrl" alt="Analysed image">
     <mat-card-content>
       <p>
-        get those labels (maybe the first 5)
+        <button mat-button *ngFor="let label of labels.slice(0, 5)">{{label}}</button>
```
Comment (asalov) [inline ics-frontend/src/app/single-image-page/single-image-page.component.html:6]: You can define the 5 label limit as a variable inside the component, so you can easily change it (if needed) from a single place

File: ics-frontend/src/app/image-classification/image-classification.component.ts [hunk 2]
```diff
@@ -37,6 +36,15 @@ export class ImageClassificationComponent {
     );}   
   }
 
+  navigateToImageClassification(imageId: number) {
```
Comment (asalov) [inline ics-frontend/src/app/image-classification/image-classification.component.ts:39]: Is this used somewhere?

File: ics-frontend/src/app/single-image-page/single-image-page.component.html (hunk 1 above)
Comment (pateva) [reply ics-frontend/src/app/single-image-page/single-image-page.component.html]: adjusted

File: ics-frontend/src/app/image-classification/image-classification.component.ts (hunk 2 above)
Comment (pateva) [reply ics-frontend/src/app/image-classification/image-classification.component.ts]: this is the first variant of the navigation I had implemented, not used anymore

File: ics-frontend/src/app/gallery/gallery.component.html [hunk 3]
```diff
@@ -2,15 +2,18 @@
...
+        <button *ngIf="searchQuery" matSuffix mat-icon-button aria-label="Clear" (click)="searchQuery=''">
           <mat-icon>close</mat-icon>
         </button>
       </mat-form-field>
     </div>
     <div class="gallery-container">
-      <div *ngFor="let imageUrl of imageUrls" class="gallery-image">
-        <img [src]="imageUrl" alt="Image">
+      <div *ngIf="images.length === 0" class="empty-handle-card">
+        <ics-overlay-nothing-found (click)="closeOverlay()"></ics-overlay-nothing-found>
```
[the remark this answers is not in the document]
Comment (pateva) [reply ics-frontend/src/app/gallery/gallery.component.html:13]: adjusted

**Pull request merged**